BUILDING.md

```diff
@@ -387,9 +387,9 @@ Prerequisites:
 * [Python 2.6 or 2.7](https://www.python.org/downloads/)
 * The "Desktop development with C++" workload from
   [Visual Studio 2017](https://www.visualstudio.com/downloads/) or the
-  "Visual C++ build tools" workload from the
+  "Build Tools for Visual Studio 2017" workload from the
```

Comment:
I think the above "Desktop development with C++" can be removed and the next link changed to `https://visualstudio.microsoft.com/downloads/` because they both redirect to that.

Reply:
@targos  - okay.  removed it as not needed.  removed that link as it looks redundant.  The link present now will take people to the build tools section of the download page.  Let me know for any more changes.

Reply:
OK thank you, that works. The link redirected me to the French version of the website which doesn't have the anchor but it works in English 👍